src/main/java/org/jitsi/jicofo/ChannelAllocator.java

```diff
@@ -95,30 +102,36 @@
     private final boolean reInvite;
 
     /**
-     * Creates new instance of <tt>ChannelAllocator</tt> which is meant to
-     * invite given <tt>Participant</tt> into the given
-     * <tt>JitsiMeetConference</tt>.
+     * A flag which indicates whether channel allocation is cancelled. Raising
+     * this makes the allocation thread discontinue the allocation process and
+     * return.
+     */
+    private boolean canceled = false;
```

Comment:
I think both are correct spellings (with the single "l" version being preferred in American English). I see in jicofo we use double "l", but in the other projects we use both spellings.